tools/Messages

```diff
@@ -53,13 +53,12 @@ Usage()
     echo ""
```

Comment:
update https://github.com/oracle/opengrok/tree/master/tools/sync if any of them use `-e or --expire`

Reply:
I was looking for that but I didn't find any. Message calls are in sync config.